products/BellHybrid/apps/application-bell-main/models/UsbStatusModel.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "UsbStatusModel.hpp"
+
+namespace app
+{
+    bool UsbStatusModel::isUsbConnected(const Store::Battery::State &state) const
+    {
+        return ((usbConnected == UsbStatus::Connected) && (state == Store::Battery::State::PluggedNotCharging)) ? true
```

Comment:
```suggestion
        return ((usbConnected == UsbStatus::Connected) && (state == Store::Battery::State::PluggedNotCharging));
```

This ternary operator is not needed here, as the comparison itself returns boolean result

Reply:
done